amnestywebsite/humanity-theme
# Remove block restrictions in PHP

Ref: N/A

In #206, we removed the block restrictions we had in place via JS, but neglected to remove the PHP implementation. This PR addresses that.

**Steps to test**:
1. disable the branding plugin, if applicable
2. whilst on `main`, attempt to insert a `core/cover` block
3. it should not be available in the editor
4. checkout this branch
5. attempt to insert a `core/cover` block again
6. you should be able to insert it
7. (don't forget to re-enable the branding plugin, if applicable)
